packages/itwinui-css/src/input/input-flex-container.scss

```diff
@@ -0,0 +1,92 @@
+// Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+// See LICENSE.md in the project root for license terms and full copyright notice.
+@mixin iui-input-flex-container {
+  --_iui-input-background-color: var(--iui-color-background);
+  --_iui-input-border-color: var(--iui-color-border-foreground);
+  --_iui-input-text-color: var(--iui-color-text);
+
+  @include iui-reset;
+  @include iui-focus($offset: -2px, $thickness: 2px);
```

Comment:
iui-focus mixin will not work here because focus is not applied on this element, it is applied on the child

Reply:
removed